app/controllers/game_controller.rb

```diff
@@ -1,35 +1,38 @@
 class GameController < ApplicationController
   skip_before_action :verify_authenticity_token
 
-  # def start
-  def initilize
-    super
-    @lives ||= 9
-    @words ||= %w[programming fun internship ruby rails coding design]
-
-    @word_to_guess ||= @words.sample
-    @hidden_word = @word_to_guess.chars.map { '_' }
+  def start
+    words = %w[programming fun internship ruby rails coding design]
+    game = GameSession.new(word_to_guess: words.sample, guesses: '')
```

Comment:
DO you think this logic belongs here, or should it live in GameSession? There's a hook I can show you for this if you want.